packages/airnode-node/src/constants.ts

```diff
@@ -22,22 +22,22 @@ export const BLOCK_MIN_CONFIRMATIONS = 0;
 export const CONVENIENCE_BATCH_SIZE = 10;
 
 // The default amount of time before a "retryable" promise is timed out and retried
-export const DEFAULT_RETRY_TIMEOUT_MS = 5_000;
+export const DEFAULT_RETRY_TIMEOUT_MS = 30_000;
 
 // The default amount of time to wait before retrying a given promise
 export const DEFAULT_RETRY_DELAY_MS = 50;
 
 // The amount of time EVM provider calls are allowed
-export const EVM_PROVIDER_TIMEOUT = 22_000;
+export const EVM_PROVIDER_TIMEOUT = 20_000;
 
 // The maximum amount of time the "initialize provider" worker is allowed before being timed out
-export const WORKER_PROVIDER_INITIALIZATION_TIMEOUT = 17_000;
+export const WORKER_PROVIDER_INITIALIZATION_TIMEOUT = 20_000;
 
 // The maximum amount of time the "process requests" worker is allowed before being timed out
-export const WORKER_PROVIDER_PROCESS_REQUESTS_TIMEOUT = 32_000;
+export const WORKER_PROVIDER_PROCESS_REQUESTS_TIMEOUT = 10_000;
```

Comment:
I find it strange that 10s is enough for this, but if the stress test says so, we can have it.

Reply:
It actually needs much less than 10s... most requests were processed in under 3s :) 
I was going to submit an update to this PR to make it 6 seconds (and give 4 more seconds to `initializeProvider`).